### ISSUE TITLE
v14 command DIR is not unicode and drops certain encoding characters.

### ISSUE THREAD

**Issue by norberturkiewicz** [the-carlisle-group/Acre-Desktop#145]
```
C:\GitHub\CarlisleGroup\CAS-6.0\APLSource\CAS3-7\?Click-2.aplf
```

vs

```
C:\GitHub\CarlisleGroup\CAS-6.0\APLSource\CAS3-7\∆Click-2.aplf
```

**Comment by norberturkiewicz** [comment]
#146 resolves this bug.

**Comment by PhilLast** [comment]
This issue _should_ be resolved by latest push `Acre 5.0.6+133 2019-02-18 16.54` where `⎕CMD dir`, indeed `⎕CMD` itself, is now used only in `buildWsFromSource` which reads only `acre/APLSource/_code-0/` that contains and is expected to continue to contain only names in `0-9`, `a-z`, `A-Z` and `_`.

**Comment by norberturkiewicz** [comment]
Fixed with the latest release 5.0.6+133
